plugins/catalog-backend/src/database/types.ts

```diff
@@ -23,6 +23,9 @@ export type DbEntitiesRow = {
   kind: string;
   name: string | null;
   namespace: string | null;
+  kind_normalized: string;
```

Comment:
Should we add them here as `| null` since they are nullable in the table?

Reply:
Was not sure if best to reflect db reality or model reality :) but sure I could change that